src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -129,7 +129,7 @@ Graph QuantizeGraph(Graph &&src) {
              mirror_node->op()->name != "_contrib_quantize")) {
           NodePtr quantize_node = InsertNode("_contrib_quantize",
             e.node->attrs.name + "_quantize", new_node, mirror_entry);
-          quantize_node->attrs.dict["out_type"] = "int8";
+            quantize_node->attrs.dict["out_type"] = "int8";
```

Comment:
why change the indent? i think the original indent is correct.

Reply:
Will change it back in next commit.

Reply:
but you shouldn't align with the line above.

Reply:
My bad eyes, will change back in the diff.